share/completions/rustup.fish

```diff
@@ -223,10 +223,8 @@ set -l __rustup_toolchains (rustup toolchain list | string replace -r "\s+.*" ''
 # but a shorter version can be used if it is unambiguous.
 set -l __rustup_toolchains_short (__rustup_strip_common_suffix_strict $__rustup_toolchains)
 
-# Needs to be global to retain access from functions
-set __rustup_channels "beta" "stable" "nightly"
-
-set rustup_profiles "minimal" "default" "complete"
+set -lx __rustup_channels "beta" "stable" "nightly"
```

Comment:
Okay, I think this actually doesn't work after all. The variable is defined, the function that would use it is defined, the variable goes out of scope, then the function is called. I think `function --inherit-variable` could be used here, but there's no real need because this is only used in a single function (`__rustup_all_toolchains`), so just define it there.

Reply:
IIRC I couldn't get them to show up prior to this so that's a win-win.